spinegeneric/cli/package_for_correction.py

```diff
@@ -137,7 +137,10 @@ def main():
 
     # Package to zip file
     print("Creating archive...")
-    fname_archive = shutil.make_archive(args.o, 'zip', path_tmp)
+    root_dir_tmp = os.path.join('/',*path_tmp.split('/')[0:-1])
```

Comment:
better os-agnostic practice: use os.pathsep

Reply:
fixed in https://github.com/spine-generic/spine-generic/pull/149/commits/457fb9e3bac143eff9d67795faaebf12564986a0